plugin/storage/integration/elasticsearch_test.go

```diff
@@ -62,6 +62,7 @@ type ESStorageIntegration struct {
 	client   *elastic.Client
 	v8Client *elasticsearch8.Client
```

Comment:
Do we still need the clients here? As long as we have clients it means we are not completely compatible with e2e tests where db info is encapsulated in the jaeger-v2 config & binary and is not part of the test itself

Reply:
yup, we can completely remove clients, but we have to refactor `es_index_cleaner_test` and `es_index_rollover_test` also.

Reply:
I see - then we can leave them for now, let's not bundle different scopes together.